sdevalapurkar/schedulearn
# update readme

# Overview

This PR updates the `README.md` file to be more accurate with the latest look and feel of the schedulearn application.

It also addresses the fact that the repository is private and that the `Running the app locally` instructions are meant for our team members and not for others to read (although at some point we may make the source code public, who knows?)

Finally, it fixes a lot of the `markdownlint` issues we previously had.

# Testing Instructions

1. Go to github.com/schedulearn and switch to my branch using the github UI
2. Here, you should see the contents of the `README.md` change and you can see the new look of it in the UI itself
3. Make sure it looks and sounds alright

# Review Requested

@Nppatel97 @CodemanDave @AhmedNSidd
